cluster-migration/src/main/java/com/wipro/ats/bdre/clustermigration/oozie/OozieMigrationPreprocessor.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.wipro.ats.bdre.clustermigration.oozie;
+
+import com.wipro.ats.bdre.clustermigration.MigrationPreprocessor;
+import com.wipro.ats.bdre.clustermigration.beans.MigrationPreprocessorInfo;
+import com.wipro.ats.bdre.exception.BDREException;
+import com.wipro.ats.bdre.util.OozieUtil;
+
+/**
+ * Created by cloudera on 3/31/16.
+ */
+public class OozieMigrationPreprocessor {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1118)